fabric-biomes-v1/src/main/java/net/fabricmc/fabric/impl/biome/FabricBiomesEntrypoint.java

```diff
@@ -0,0 +1,35 @@
+package net.fabricmc.fabric.impl.biome;
+
+import com.mojang.serialization.Lifecycle;
+
+import net.minecraft.util.registry.MutableRegistry;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.util.registry.RegistryKey;
+import net.minecraft.world.biome.Biome;
+
+import net.fabricmc.api.ModInitializer;
+import net.fabricmc.fabric.api.biomes.v1.FabricBiomeBuilder;
+import net.fabricmc.fabric.api.biomes.v1.event.BiomeLoadingCallback;
+import net.fabricmc.fabric.api.event.registry.DynamicRegistryEntryAddedCallback;
+
+public class FabricBiomesEntrypoint implements ModInitializer, DynamicRegistryEntryAddedCallback<Biome> {
```

Comment:
This entire PR is a really hard sell for me if it depends on another PR that has not been merged yet.

I don't see any real reason to review until the backing logic is merged.

Gonna have to stop my review here.

Reply:
That's unfortunate. But this is a draft after all, so I don't expect full reviews.
I'm just hoping on some input.